src/cryptoadvance/specter/specter.py

```diff
@@ -156,6 +156,14 @@ def cleanup_on_exit(self, signum=0, frame=0):
             if not node.external_node:
                 node.stop()
 
+        if (
+            self.service_manager.get_service("notifications")
+            and self.service_manager.get_service("notifications").notification_manager
+        ):
+            self.service_manager.get_service(
+                "notifications"
+            ).notification_manager.quit()
+
```

Comment:
I think that should be implemented in a new callback inside the extension(s). What about `callbacks.cleanup_on_exit` ?!

Reply:
Good point. I will do that.

Reply:
Done in  acba2fa5